sync/ui.md

```diff
@@ -9,6 +9,12 @@ As the server will be running on OpenShift, a ServiceAccount Oauth Client will b
 This follows the pattern in the [Auth proposal here](../auth/developer-single-sign-on-across-mobile-services.md).
 Any OpenShift user who has 'edit' access to the the `Deployment` object for the Data Sync server will be able to get access to the Data Sync Admin UI.
 
+## Architecture
+
+The Admin UI will be located in the same repository as the Server. It will use React with JSX and the JSX will be compiled using [babel](https://reactjs.org/docs/add-react-to-a-website.html#add-jsx-to-a-project). We want to
+avoid the use of heavyweight build tools (Webpack, Grunt, Gulp) as much as possible and rely on npm scripts. For routing the most popular option (for web projects) is [react-router](https://github.com/ReactTraining/react-router). We can investigate others but i would suggest to use this library.
```

Comment:
I wouldn't mind using Webpack eventually but for starters I agree we stick with Babel.